build/yamls/windows/containerd-with-ovs/conf/Run-AntreaOVS-Containerd.ps1

```diff
@@ -2,11 +2,26 @@ $ErrorActionPreference = "Stop"
 $mountPath = $env:CONTAINER_SANDBOX_MOUNT_POINT
 $mountPath = ($mountPath.Replace('\', '/')).TrimEnd('/')
 $env:PATH = $env:PATH + ";$mountPath/Windows/System32;$mountPath/openvswitch/usr/bin;$mountPath/openvswitch/usr/sbin"
+$OVSDriverDir="$mountPath\openvswitch\driver"
+$WorkDir="$mountPath\openvswitch\"
+
+# Check if OVSExt driver is already installed
+if (!(netcfg -q ovsext)) {
```

Comment:
Here is not updated. I think you should use this workflow for the change: 1) update this direct powershell script, 2) run `make manifest` to auto-generate antrea-windows-containerd-with-ovs.yml. If the yml file is not generated, check the configurations `.PHONY: manifest` in Makefile

Reply:
Yes I have updated the Run-Antrea-OVS-Containerd.ps1 script and followed the workflow you mentioned